retroshare-gui/src/gui/gxschannels/CreateGxsChannelMsg.cpp

```diff
@@ -797,15 +800,16 @@ void CreateGxsChannelMsg::sendMessage(const std::string &subject, const std::str
 		QByteArray ba;
 		QBuffer buffer(&ba);
 
-        RsGxsImage image;
+		RsGxsImage image;
+		bool has_transparency = ImageUtil::hasAlphaContent(picture.toImage());
```

Comment:
you test "picture" for transparency, but what is actually saved is getCropScaledPicture(). There's no reason the transparency is kept there!

Reply:
so save the result of preview_W->getCroppedScaledPicture() into a QImage qimg, and check the alpha of qimg